opentelemetry/proto/common/v1/common.proto

```diff
@@ -72,6 +72,8 @@ message StringKeyValue {
 // InstrumentationLibrary is a message representing the instrumentation library information
 // such as the fully qualified name and version. 
 message InstrumentationLibrary {
+  // An empty instrumentation library name is equivalent with an "unknown" 
```

Comment:
I think this is confusing.It is not literally equivalent to to a library with the name equal to string "unknown", right? Can we remove the double quote characters and just say:
```suggestion
  // An empty instrumentation library name means the name is unknown. 
```